cargo-apk/README.md

```diff
@@ -123,6 +123,11 @@ launch_mode = "singleTop"
 # Defaults to "unspecified".
 orientation = "landscape"
 
+# See https://developer.android.com/guide/topics/manifest/activity-element#exported
+#
+# Defaults to "unspecified".
```

Comment:
The page you link just above states that the default is considered `false` when it is unset/unspecified.

I'm not sure if we should have this as tri-state, @maxded what do you think? We can effectively set this (and the other booleans in the manifest to):
- Unset, nothing is emitted in the XML: results in whatever Android says/thinks is the default;
- `false`: explicitly set to false, we could let the serializer default to this if the docs say it's the default when unspecified;
- `true`: explicitly set to true, same story for the default as above.

Reply:
I left it unspecified by default to avoid breaking anything relying on the actual default manifest state. 

I think it might be a good idea to let it unspecified by default and let android versions prior to 12 handle it just the way it did before (considered true when any `IntentFilter` is present) and force the user to make the choice for android 12.

Reply:
If unset does not break the default manifest of android versions prior to 12, which I don't think it does, I _think_ unset is the best option here. That way a user is guaranteed that the android documentation is their source of truth and means it is less error-prone on our side.

Reply:
Fair enough, leaving it unset and keeping it out of the manifest by default (ie `None`) is our best option here.

> and force the user to make the choice for android 12.

I disagree as by far most android-ndk-rs use-cases are single-activity native applications that are supposed to be started externally, see the comment below. When ndk-build provides the `MAIN/LAUNCHER` intent (because the user hasn't set it) we should also make it `exported=true` for them.

In any case, this comment was initially placed to request a change on the `# Defaults to "unspecified".` - `screenOrientation` defaults to the _string_ "unspecified" - this boolean cannot default to a string. How about documenting that it is a boolean that is unset (or set to `true` if the default `MAIN` intent is provided), and also inheriting some of the AOSP docs explaining that this is mandatory Since Android 12?

Reply:
> I disagree as by far most android-ndk-rs use-cases are single-activity native applications that are supposed to be started externally, see the comment below. When ndk-build provides the MAIN/LAUNCHER intent (because the user hasn't set it) we should also make it exported=true for them.

Ok, it makes sense, I'll change that.

> In any case, this comment was initially placed to request a change on the # Defaults to "unspecified". - screenOrientation defaults to the string "unspecified" - this boolean cannot default to a string. How about documenting that it is a boolean that is unset (or set to true if the default MAIN intent is provided), and also inheriting some of the AOSP docs explaining that this is mandatory Since Android 12?

My bad I read "unspecified" as "unset" I'll change that too. Yeah I agree on with adding the documentation explaining that it is now required.